internal/run/convert.go

```diff
@@ -88,7 +158,22 @@ func (c *Convert) Execute() error {
 		Context: c.kubeContext,
 	}
 
-	return convertcmd.Convert(c.convertOptions, kc)
+	if err := convertcmd.Convert(c.convertOptions, kc); err != nil {
+		return err
+	}
+
+	clientset, err := clientsetFromFile(c.kubeConfig)
```

Comment:
will do that although formally this error is handled by convert too: https://github.com/helm/helm-2to3/blob/master/pkg/v2/release.go#L126

I copy paste the code but instead using a clientset as parameter to help on testing.

Reply:
Why not just use the upstream API from 2to3 https://pkg.go.dev/github.com/helm/helm-2to3/pkg/v2#GetReleaseVersions rather than copy and past code?

Reply:
it's possible but we won't be able to unit test as it depend on a kubeconfig file

Reply:
Gotcha, this is because the upstream code isn't unit tested.

However, I will point out that we already trust this code when we run `convertcmd.Convert` since it uses this same API.

Reply:
other reason, and actually the most important one, is that the upstream code returns Release data and not metadata information related to the Configmap itself, and we need the later as this code is modifying a label without really caring about the Release information contained